bindings/bind_py_structure_manager.cc

```diff
@@ -458,6 +458,132 @@ namespace rascal {
         py::call_guard<py::gil_scoped_release>());
   }
 
+  /**
+   * Bind a getter for the feature matrix comming from BlockSparseProperty.
+   *
+   * it will return a dictionary with keys->atomic number vals->feature mat
+   * the feature matrices size is (n_centers, inner_size)
+   */
+  template <class Calculator, class ManagerCollection_t,
+            class ManagerCollectionBinder>
+  void bind_sparse_feature_matrix_getter(
+      ManagerCollectionBinder & manager_collection) {
+    manager_collection.def(
+        "get_sparse_feature_matrix",
+        [](ManagerCollection_t & managers, Calculator & calculator) {
+          using Manager_t = typename ManagerCollection_t::Manager_t;
+          using Prop_t = typename Calculator::template Property_t<Manager_t>;
+          using Keys_t = typename Prop_t::Keys_t;
+          using VecMap_t = Eigen::Map<math::Vector_t>;
+
+          auto property_name{managers.get_calculator_name(calculator, false)};
+
+          auto && property_ =
+              managers[0]->template get_property_ref<Prop_t>(property_name);
```

Comment:
is `managers` guaranteed to have at least 1 element? Else you could use `managers.at(0)-> ...`

Reply:
added checks